src/web/components/CommentsLayout.stories.tsx

```diff
@@ -31,7 +31,9 @@ export const Default = () => (
                 discussionApiClientHeader="testClientHeader"
                 expanded={false}
             />
-            <RightColumn>{/* TODO: Comments ad slot goes here */}</RightColumn>
+            <RightColumn>
+                <AdSlot asps={namedAdSlotParameters('comments')} />
```

Comment:
```suggestion
                {/* This is the space where the comments ad slot would go */}
```

Reply:
Thanks @oliverlloyd it was my fault. I pushed too early :)